WorkloadManagementSystem/Agent/JobAgent.py

```diff
@@ -273,6 +277,12 @@ def execute( self ):
     if 'CPUTime' not in params:
       self.log.warn( 'Job has no CPU requirement defined in JDL parameters' )
 
+    # Job requirement for a number of processors
+    processors = 1
+    if 'NumberOfProcessors' in params:
+      processors = int( params['NumberOfProcessors'] )
+    wholeNode = True if 'WholeNode' in params else False
```

Comment:
In one case this is a CS option name that we start with a capital by convention. In the other case this is a keyed function argument which we start with lower case